src/App.scss

```diff
@@ -1 +1,84 @@
-// styles go here
+$mainColor: rgb(142, 162, 226);
+$hoverListElementColor: rgb(216, 216, 216);
+$titleColor: rgb(127, 136, 255);
+
+body {
+  box-sizing: border-box;
```

Comment:
if you want to add `box-sizing: border-box;` to all elements it's better to use * selector, not body